ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -488,8 +488,8 @@ private void updateImeOptions() {
 
   @Override
   protected boolean verifyDrawable(Drawable drawable) {
-    if (mContainsImages && getText() instanceof Spanned) {
-      Spanned text = (Spanned) getText();
+    if (mContainsImages) {
```

Comment:
As for this class, you could see that checks are not required as TextEdit always returns Editable which is an instance of Spanned.